core/domain/exp_jobs_one_off_test.py

```diff
@@ -953,6 +953,64 @@ def setUp(self):
         self.albert_id = self.get_user_id_from_email(self.ALBERT_EMAIL)
         self.process_and_flush_pending_tasks()
 
+    def create_exploration_with_states_schema_version(
```

Comment:
Maybe add a note that the caller needs to make sure that the passed-in states_dict is compatible with the passed-in state_schema_version? This seems like an important constraint.

Also, in some places you use states_schema_version and in others you use state_schema_version. Please keep the docstring, varnames etc. consistent with whatever the correct representation is for an exploration in the datastore.

Reply:
Done